Include context variables.

charginghawk opened the pull request:
While the breakpoint works, none of the variables from the Twig template are available to inspect. Setting 'needs_context' to true will pass the variables to setBreakpoint(), which in turn needs to accept the variables as an argument so they appear through xdebug.

ajgarlag commented:
Looks great!

Can you rename `$variables` to `$context` and add the `@param` annotation to the `setBreakpoint` docblock?

charginghawk commented:
Fixed per recommendations - I also added in environment information, since it often goes hand in hand with context. Please let me know if there's any other issues.

If you're interested, I plan on using this to power a Drupal module for xdebug breakpoints in Twig:
https://www.drupal.org/sandbox/charginghawk/2697135

Thanks!

ajgarlag commented:
Good to know about the drupal module.

Please, squash the PR to merge it.

charginghawk commented:
Squashed

ajgarlag commented:
Thank you @charginghawk.
